projects/demo/src/modules/components/input-tag/input-tag.template.html

```diff
@@ -84,6 +84,15 @@
 
             <tui-input-tag-example-8></tui-input-tag-example-8>
         </tui-doc-example>
+
+        <tui-doc-example
+            id="mask"
+            heading="Mask"
+            [content]="example9"
+            [fullsize]="true"
+        >
+            <tui-input-tag-example-9></tui-input-tag-example-9>
```

Comment:
Add `max-width` for new example

<img width="1768" alt="Screenshot 2023-10-16 at 16 36 53" src="https://github.com/taiga-family/taiga-ui/assets/35179038/e12362b4-7a78-40db-ac6a-3cdbeb070bd5">